### PR TITLE
fixup unit test description

### PR DESCRIPTION
**Pull request by perdasilva**
This PR just addresses a comment in a previous PR where a unit test description was found to be mismatched

Signed-off-by: perdasilva <perdasilva@redhat.com>

### CODE REVIEW

File: internal/util/logging.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package util
```
Comment (tmshort) [inline internal/util/logging.go:1]: Is this used?

Comment (tmshort) [reply internal/util/logging.go:1]: (meaning the whole file)

Comment (perdasilva) [reply internal/util/logging.go:1]: oh crap - nah that snuck in

Comment (perdasilva) [reply internal/util/logging.go:1]: was also trying to think about logging then gave up - ty!

**Review by awgreene — APPROVED** [review]
/approve

**Pull request merged**